core/src/main/java/com/linecorp/armeria/internal/client/HttpSession.java

```diff
@@ -113,4 +118,7 @@ void invoke(PooledChannel pooledChannel, ClientRequestContext ctx,
     void deactivate();
 
     int incrementAndGetNumRequestsSent();
+
+    @UnstableApi
+    void initiateConnectionShutdown();
```

Comment:
It's an internal API, which doesn't require `@UnstableApi`. 
```suggestion
    void initiateConnectionShutdown();
```